sigtool/vba.c

```diff
@@ -891,25 +899,26 @@ static void wm_decode_macro(unsigned char *buff, uint32_t len, int hex_output)
             case 0x79: /* unicode "string" */
                 w_length = (uint16_t)(buff[i + 2] << 8) + buff[i + 1];
                 tmp_buff = (unsigned char *)malloc((w_length * 2) + 1);
+                if (!tmp_buff) return;
                 memcpy(tmp_buff, buff + i + 3, w_length * 2);
                 tmp_name = (unsigned char *)get_unicode_name((char *)tmp_buff, w_length * 2);
                 free(tmp_buff);
                 printf("\"%s\"", tmp_name);
                 free(tmp_name);
                 i += 3 + (w_length * 2);
                 break;
-
             case 0x7c: /* unicode 'string */
                 s_length = (uint8_t)buff[i + 1];
                 tmp_buff = (unsigned char *)malloc((s_length * 2) + 1);
+                if (!tmp_buff) return;
                 memcpy(tmp_buff, buff + i + 2, s_length * 2);
                 tmp_name = (unsigned char *)get_unicode_name((char *)tmp_buff, s_length * 2);
                 free(tmp_buff);
                 printf("'%s", tmp_name);
                 free(tmp_name);
                 i += 2 + (s_length * 2);
                 break;
-
+                break;
```

Comment:
```suggestion
                break;
```

It looks like an extra `break;` was accidentally added.